.github/workflows/jstest.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  workflow_call:
+    secrets:
+      JSTEST_PRIVATE_SSH_KEY:
+        description: 'Private key to access the jstest repo'
+        required: true
+
+jobs:
+  jstest:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: "./packages/pages"
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.4
+
+      - name: Build with NodeJS 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "pnpm"
+
+      - name: Install deps
+        run: pnpm i
+
+      - name: Pack
+        run: pnpm pack --pack-destination latest
+
+      - name: Checkout JSTEST
```

Comment:
This checks out jstest, but how does it run the tests of each package using the local packed version of pagesjs?

Reply:
It doesn't yet, still WIP

Reply:
It installs and attempts to run playwright in all the packages now, but am running into some webserver errors that don't occur locally: https://github.com/yext/pages/actions/runs/6527694680/job/17722896253?pr=405

Debugging..